server/views/topics/story.py

```diff
@@ -28,7 +28,7 @@ def story(topics_id, stories_id):
             if tag['tag_sets_id'] == tag_util.GEO_TAG_SET:
                 geonames_id = int(tag['tag'][9:])
                 try:
-                    tag['geoname'] = _cached_geoname(geonames_id, story['language'])
+                    tag['geoname'] = _cached_geoname(geonames_id, story_info['language'])
```

Comment:
I want to confirm that `story_info` is the correct object to use here. I think the other candidate is `story_topic_info`. I inspected both objects in the debugger and they both have a "language" attribute.

Reply:
I see! I agree that updating line 27 makes that clearer too.